libs/components/datetime/src/lib/modules/timepicker/timepicker.component.ts

```diff
@@ -120,18 +118,18 @@ export class SkyTimepickerComponent implements OnInit, OnDestroy {
     return moment(this.activeTime).minute() + 0;
   }
 
-  public set selectedTime(newTime: SkyTimepickerTimeOutput) {
+  public set selectedTime(newTime: SkyTimepickerTimeOutput | undefined) {
```

Comment:
This is not an input, so can remain as-is.

Reply:
This is set to the value of `#modelValue` within the directive which can be `undefined` so it needed updated.